pystemd/run.py

```diff
@@ -28,6 +28,11 @@
 EXIT_SUBSTATES = (b"exited", b"failed", b"dead")
 
 
+class DynamicUser:
```

Comment:
can you move this to `https://github.com/facebookincubator/pystemd/blob/master/pystemd/utils.py` "from pystemd.run import DynamicUser" should not work, if it works (i'm away from a real interpreter to try) its due the weird __module__ hack at the end of this file.